backend/app/models/concerns/enum_model.rb

```diff
@@ -31,7 +31,16 @@ def find_many(slugs)
     end
 
     def find(slug)
-      all.find { |o| o.slug == slug }
+      new(slug: slug)
+    end
+
+    def select_index_sql(column_name = name.underscore)
```

Comment:
need to try this, because this is kind of advanced :D. At first sight `select_index_sql` is class method, but `name` is instance method so not sure how this works out...

Reply:
btw not sure, but this would be probably worthy test so I don't need to test it manually :D

Reply:
Ok, I will add some test. `name` is a class.name.